packages/lib/shim-init-node.ts

```diff
@@ -571,6 +583,11 @@ function shimInit(options: ShimInitOptions = null) {
 					});
 
 					const request = http.request(requestOptions, (response: any) => {
+
+						if (downloadController !== null) {
```

Comment:
No need for strict null checking here. What you want to check is if it's defined or not and for that just `if (downloadController)` is enough. There are places maybe where we need to make the difference between undefined and null or other falsy values, but not when we just need to know if something exists or not.